doc/guides/collaborator-guide.md

```diff
@@ -87,6 +88,33 @@ to land but is [author ready](#author-ready-pull-requests), add the
 `author ready` label. If you wish to land the pull request yourself, use the
 "assign yourself" link to self-assign it.
 
+### Managing security issues
+
+Security issues should ideally be reported through the processes outlined in
+[SECURITY.md][security reporting]. This allows the collaborators to
+appropriately triage the report and address vulnerabilities in a planned
+security release. If an issue is opened in the public repo
+which describes a security issue, or if an issue is later identified to be
+describing a security issue, take the following steps:
+
+* Ask the originator to submit a report through Hacker one as outlined in
+  [SECURITY.md][security reporting].
+* Move the issue to the private repo called `premature-disclosures`
```

Comment:
Link to the repository? Or should we keep it without a link on purpose?

Also, I assume it will be a repo on nodejs-private?

Reply:
I've not created the repo yet, but a link makes sense. It should only be accessible to those who have access to private repos within the org

Reply:
@mmarchini added the link in the first reference, not sure if we need to make all references a link or not.

Reply:
> Also, I assume it will be a repo on nodejs-private?

No, its a private repo in the nodejs org as I don't believe we can move issues across organizations.

Reply:
I see. The downside is that folks who have access to private repos in this org but not on the repo we usually use for security releases will have access to the issue. It's _probably_ fine though, it only means some folks in moderation and CommComm will have access to the issue even when they don't have access to security release discussions (which is still better than keeping the issue public).

Reply:
@mmarchini 
> which is still better than keeping the issue public

+1